src/components/LeftBar.vue

```diff
@@ -319,13 +317,17 @@ export default {
   box-shadow: 0 3px 16px -7px rgb(17 18 24 / 70%);
 }
 
+.leftbar-top {
```

Comment:
Since you decided to a new field called `leftbar-top`, please add a 10-15px margin to the top so the left bar looks nicer.

Before:
![image](https://github.com/Lazap-Development/Lazap/assets/59381835/47834cc4-ce81-42cc-a188-299b218f4c78)

After:
![image](https://github.com/Lazap-Development/Lazap/assets/59381835/29c6d01a-fd3b-4058-b24c-5ebb7530f671)

Reply:
Added margin to the leftbar top div